service/telemetry/config.go

```diff
@@ -102,4 +102,7 @@ type MetricsConfig struct {
 
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
+
+	// NameSpace indicates the telemetry namespace the service will set.
+	NameSpace string `mapstructure:"namespace"`
```

Comment:
I'd recommends going with `Namespace` instead, since it is a single word.

Reply:
sounds good. will update